crates/moss-theme/theme_util.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+from pathlib import Path
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Generate Rust Constants for JSON Schema ")
+    parser.add_argument("--schema", type=argparse.FileType('r'), required=True)
+    parser.add_argument("--output", type=argparse.FileType('w+'), required=True)
+    args = parser.parse_args()
+
+    schema_file = args.schema
+    output_file = args.output
+
+    schema = schema_file.read()
+
+    output_code = \
+        f"""
+use serde_json::{{json, Value}};
+        
+pub const SCHEMA_{Path(schema_file.name).stem.upper()}: Value = json!({schema}
+);
+"""
```

Comment:
This needs to be abstracted into a separate function because, in the future, we may need to add more logic. We shouldn’t clutter the entry point with this.

```python 
def generate_rust_constant(schema_name: str, schema: dict) -> str:
    """
    Generate Rust code for a constant containing the JSON schema.
    """
    return f"""\
use serde_json::{{json, Value}};

pub const SCHEMA_{schema_name.upper()}: Value = json!(
{json.dumps(schema, indent=4)}
);
"""
```